Script.js

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<body>
+
+<h2>JavaScriptExercise</h2>
+
+<script>('.col-sm-4').on('click',red(){  })</script>
```

Comment:
This isn't quite there yet. Have you tested your code? Your selector doesn't seem to be correct, and there's no such function as red(). Here are the API docs for the `$.on` function. http://api.jquery.com/on/. It takes two arguments, an event, such as `click`, and a function describing what to do when clicking.